src/js/rebuses.js

```diff
@@ -631,3 +631,5 @@ export function getRebuses() {
     };
   });
 }
+
+export const totalRebusesCount = rebuses.length;
```

Comment:
If we use the props passed to the render method instead in the ProgressBar component, we don't need to export/import this 🙂